docs/nlu/components.rst

```diff
@@ -39,9 +32,9 @@ MitieNLP
 :Outputs: Nothing
 :Requires: Nothing
 :Description:
-    Initializes mitie structures. Every mitie component relies on this,
+    Initializes MITIE structures. Every MITIE component relies on this,
     hence this should be put at the beginning
-    of every pipeline that uses any mitie components.
+    of every pipeline that uses any MITIE components.
```

Comment:
What happens if you forget to specify this? Do you get an error?

Reply:
Yes. If a required component is not in your pipeline, you will get an error pointing you to the missing component.